tests/instruments/oxfordinstruments/test_ps120_10.py

```diff
@@ -0,0 +1,80 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2023 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import pytest
+
+from pymeasure.test import expected_protocol
+
+
+from pymeasure.instruments.oxfordinstruments.ps120_10 import PS120_10
+
+
+def test_version():
+    with expected_protocol(PS120_10,
+                           [("V", "IPS120-10 Version 3.04 @ Oxford Instruments 1996")]
+                           ) as inst:
+        assert inst.version == "IPS120-10 Version 3.04 @ Oxford Instruments 1996"
+
+
+def test_activity_getter():
+    with expected_protocol(PS120_10,
+                           [("X", "X00A0C0M00P00")]
+                           ) as inst:
+        assert inst.activity == "hold"
+
+
+def test_activity_setter():
+    with expected_protocol(PS120_10,
+                           [("A0", "A")]
+                           ) as inst:
+        inst.activity = "hold"
+
+
+@pytest.mark.xfail()  # TODO this test fails, the correct response has to be added.
+def test_current_setpoint_getter():
+    with expected_protocol(PS120_10,
+                           [("R0", "R+130")]
```

Comment:
```suggestion
                           [("R0", "R+00130")]
```

Reply:
I already ajusted the test according to your log entries, but that is not the issue.